app/misc.py

```diff
@@ -229,6 +232,15 @@ def get_global_stylesheet(self):
         return ''
 
 
+def is_target_user_admin(uid):
+    try:
+        check_admin = UserMetadata.get((UserMetadata.uid == uid) and (UserMetadata.key == 'admin') and (UserMetadata.value == '1'))
+        if check_admin:
+            return True
+        return False
```

Comment:
The last return is a bit redundant. If it gets past the `.get()` without throwing any exception you're guaranteed to have a record in `check_admin` (if you apply this change you should remove the `check_admin` variable, since it would leave it unused)

```suggestion
        return True
```